test/docker-compose.mariadb.yml

```diff
@@ -0,0 +1,220 @@
+version: "2"
+
+services:
+  # https://docs.docker.com/docker-hub/builds/automated-testing/
+  sut:
+    build:
+      context: ./
+      dockerfile: Dockerfile_test
+    command: sh /docker_test.sh
+    # Only for travis-ci, do not name test container in dockerhub
+    container_name: sut
+    depends_on:
+      - erpnext_db
+      - erpnext_app
+      - erpnext_web
+      - erpnext_scheduler
+      - erpnext_worker_default
+      - erpnext_worker_long
+      - erpnext_worker_short
+      - erpnext_socketio
+      - erpnext_redis_cache
+      - erpnext_redis_queue
+      - erpnext_redis_socketio
+    environment:
+      # Container setup
+      - NODE_TYPE=test
+      # Docker setup
+      - DOCKER_APPS_TIMEOUT=900
+      - DOCKER_DEBUG=1
+    volumes_from: 
+      - erpnext_app
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - /etc/timezone:/etc/timezone:ro
+
+  erpnext_app:
+    build:
+      context: ./
+      dockerfile: Dockerfile.${VARIANT}
+      args:
+          - BUILD_BRANCH=${TRAVIS_BRANCH}
+          - BUILD_URL=https://github.com/${TRAVIS_REPO_SLUG}
+    image: ${IMAGE_NAME}
+    container_name: erpnext_app
+    command: app
+    #restart: always
+    ports:
+      - 8000:8000
+    depends_on:
+      - erpnext_db
+    links:
+      - erpnext_db
+    environment:
+      # Docker setup
+      - DOCKER_DB_ALLOWED_HOSTS=
+      - DOCKER_APPS_TIMEOUT=900
+      # Frappe setup
+      - FRAPPE_APP_INIT=erpnext erpnext_autoinstall
+      - FRAPPE_DEFAULT_PROTOCOL=http://
+      - FRAPPE_DEFAULT_SITE=${ERPNEXT_SITE}
+      - FRAPPE_LOGGING=1
+      - DEVELOPER_MODE=0
+      #- Amin user setup
+      - ADMIN_PASSWORD=${ERPNEXT_ADMIN_PWD}
+      - ENCRYPTION_KEY=${ERPNEXT_ENCRYPTION_KEY}
+      # Database setup
+      - DB_TYPE=mariadb
+      - DB_HOST=erpnext_db
+      - DB_PORT=3306
+      - DB_NAME=${ERPNEXT_DB_NAME}
+      - DB_PASSWORD=${ERPNEXT_DB_PWD}
+      - DB_ROOT_PASSWORD=${ERPNEXT_DB_ROOT_PWD}
+      # Mail setup
+      - MAIL_MUTED=true
```

Comment:
@AminovE99 We need to add the environment values to check that the autoinstall works when configuration is defined.
We might need to define a second docker-compose file (one with the config and one without).